modules/@apostrophecms/schema/ui/apos/components/AposInputRange.vue

```diff
@@ -0,0 +1,248 @@
+      margin-left: 5px;
+    }
+  }
+
+  .apos-range-input {
+    flex-grow: 1;
+    margin-right: 20px;
+  }
+
+  .apos-range-input__scale {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 5px;
+  }
+
+  .apos-range-input__scale {
+    @include type-small;
+    color: var(--a-base-4);
+    transition: color 0.5s ease;
+  }
+
+  // adapted from http://danielstern.ca/range.css/#/
+  .apos-range {
+    width: 100%;
+    margin: 5px 0;
+    background-color: transparent;
+    /* stylelint-disable-next-line property-no-vendor-prefix */
+    -webkit-appearance: none;
+    transition: all 0.3s ease;
+    &:focus {
+      outline: none;
+      & + .apos-range-input__scale {
+        color: var(--a-text-primary);
+      }
+    }
+  }
+
+  .apos-range::-webkit-slider-runnable-track {
+    width: 100%;
+    height: 5px;
+    border: 1px solid var(--a-base-4);
+    background: var(--a-base-7);
+    border-radius: 25px;
+    cursor: pointer;
+  }
+
+  .apos-range::-webkit-progress-value {
+    background: var(--a-primary);
+  }
+
+  .apos-range::-webkit-slider-thumb {
+    margin-top: -6px;
+    width: 15px;
+    height: 15px;
+    border: 1px solid var(--a-primary-button-active);
+    border-radius: 50%;
+    background: var(--a-primary);
+    cursor: pointer;
+    /* stylelint-disable-next-line property-no-vendor-prefix */
+    -webkit-appearance: none;
+  }
+
+  .apos-range:focus::-webkit-slider-runnable-track {
+    border: 1px solid var(--a-base-3);
+    background: var(--a-base-6);
+  }
+
+  .apos-range::-moz-range-track {
+    border-radius: 25px;
+    width: 100%;
+    height: 5px;
+    border: 1px solid var(--a-base-4);
+    background: var(--a-base-7);
+    cursor: pointer;
+  }
+
+  .apos-range::-moz-range-thumb {
+    width: 15px;
+    height: 15px;
+    border: 1px solid var(--a-primary-button-active);
+    background: var(--a-primary);
+    border-radius: 50%;
+    cursor: pointer;
+  }
+
+  .apos-range::-ms-track {
+    width: 100%;
+    height: 5px;
+    color: transparent;
+    background: transparent;
+    border-color: transparent;
+    border-width: 5.8px 0;
+    cursor: pointer;
+  }
+
+  .apos-range::-ms-fill-lower {
+    border: 1px solid var(--a-base-4);
+    border-radius: 50px;
+    background: var(--a-base-7);
+  }
+
+  .apos-range::-ms-fill-upper {
+    border: 1px solid var(--a-base-4);
+    background: var(--a-base-7);
+    border-radius: 50px;
+  }
+
+  .apos-range::-ms-thumb {
+    width: 15px;
+    height: 15px;
+    border: 1px solid var(--a-primary-button-active);
+    border-radius: 1px;
+    background: var(--a-primary);
+    cursor: pointer;
+    margin-top: 0;
+    /*Needed to keep the Edge thumb centered*/
+  }
+
+  .apos-range:focus::-ms-fill-lower {
+    background: var(--a-base-7);;
+  }
+
+  .apos-range:focus::-ms-fill-upper {
+    background: var(--a-base-7);;
+  }
+
+  /*TODO: Use one of the selectors from https://stackoverflow.com/a/20541859/7077589 and figure out
```

Comment:
Do we need to fuss with little alignment bits in IE11?

Reply:
axed